src/control-plane/backend/lambda/api/service/reporting.ts

```diff
@@ -917,7 +918,11 @@ export class ReportingService {
       logger.info('end of warm up reporting service');
       return res.status(201).json(new ApiSuccess(dashBoards.DashboardSummaryList));
     } catch (error) {
-      next(`Warmup redshift serverless with error: ${error}`);
+      logger.warn(`Warmup redshift serverless with error: ${error}`);
+      if ( error instanceof ValidationException || error instanceof STSServiceException) {
```

Comment:
Why are we using dataRole from the client? For performance optimization?

Reply:
delete redshift info in warmup api parameter